packages/homepage/content/articles/new-dashboard/index.md

```diff
@@ -0,0 +1,93 @@
+---
+banner: ./images/banner.png
+slug: new-dashboard
+authors: ["Ives van Hoorne"]
+photo: https://avatars0.githubusercontent.com/u/587016?s=460&v=4
+title: A New-look Dashboard
+description:
+  Our new dashboard will get you started faster and keep your code organized.
```

Comment:
Same for this, I don't think the new dashboard should be the main story. I think it should be around how we're making CodeSandbox more collaborative by introducing a place where people can manage and organize their work together.